undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -108,6 +108,8 @@ undertow.ajp-listener.remove=Removes an AJP listener
 undertow.access-log=The access log configuration for this virtual server.
 undertow.access-log.add=Add access log configuration to this virtual server.
 undertow.access-log.remove=Remove access log configuration from this virtual server.
+undertow.access-log.log-to-file=If the access log messages should be written to the file. Note this attribute is \
+  mutually exclusive with the use-server-log attribute.
```

Comment:
Is this really used? I don't see it as an attribute in `access-log` element.

Reply:
Nope, good catch. I'll get that removed thanks!